docs/DesignDocs/system_ocall_opt_in.md

```diff
@@ -23,6 +23,14 @@ Users can import a system OCall the same way any other EDL file would be importe
 if an enclave with EDL file `sample.edl` wants to use the function `oe_log_ocall()` they can do
 one of the following:
 
+*Note: SGX SDK requires the user to link specific libraries in addition to importing the system edl.
```

Comment:
Should we say Intel SGX SDK?

Reply:
I thought I heard in Sig-Arch that Intel SGX SDK has been contributed to CCC and is just SGX SDK.
CC: @dthaler

Reply:
Technically there's two different SDK's right?   There's the "SGX SDK for Linux" that is now a CCC project with that label on the CCC site.  And then there's the "Intel SGX SDK for Windows".   If this sentence applies to both, then maybe it should explicitly mention both, rather than incorrectly implying there's only one.

Reply:
Mentioned both SDKs.